activerecord/lib/rails/generators/active_record/migration.rb

```diff
@@ -40,7 +40,7 @@ def default_migrate_path
         end
 
         def configured_migrate_path
-          return unless database = options[:database]
+          database = options.fetch(:database, ActiveRecord.migration_generator_default_database)
```

Comment:
I am not simplifying the `config&.migrations_path` a few lines below, as code needs to be resilient to users making typos when tweaking the default database name.

However, a better handling of _non-existent provided DB names_ is advisable, otherwise, developers could think that they are creating schema-changes for a particular DB, when in truth they are getting fall-backed to the DB associated to `db/migrate` path.